Module 7/ex0/Includes/Whatever

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+
+template <typename T> class Whatever {

```

Comment:
What was your intention behind the creation of this class ?
You don't use it anywhere, you can remove it.